rustls/src/crypto/aws_lc_rs/tls13.rs

```diff
@@ -220,6 +248,57 @@ impl MessageDecrypter for AeadMessageDecrypter {
     }
 }
 
+struct GcmMessageEncrypter {
+    enc_key: aead::TlsRecordSealingKey,
+    iv: Iv,
+}
+
+impl MessageEncrypter for GcmMessageEncrypter {
+    fn encrypt(&mut self, msg: BorrowedPlainMessage, seq: u64) -> Result<OpaqueMessage, Error> {
+        let total_len = msg.payload.len() + 1 + self.enc_key.algorithm().tag_len();
```

Comment:
I think this will need to be caught up with https://github.com/rustls/rustls/pull/1579 once it lands.

Reply:
rebased